build/lin.sh

```diff
@@ -340,10 +416,12 @@ printf "\"${PLATFORM}\"" >platform.json
 # Add third-party notices
 curl -Os https://raw.githubusercontent.com/lovell/sharp-libvips/master/THIRD-PARTY-NOTICES.md
 
-# Create .tar.gz
-tar czf /packaging/libvips-${VERSION_VIPS}-${PLATFORM}.tar.gz \
+# Create the tarball
+rm -rf lib
+mv lib-filterd lib
+tar chzf ${PACKAGE}/libvips-${VERSION_VIPS}-${PLATFORM}.tar.gz \
```

Comment:
The `tar czf` -> `tar chzf` change dereferences symbolic links during packing, which could be useful for https://github.com/lovell/sharp/issues/2197.